relay.go

```diff
@@ -544,12 +549,12 @@ func (r *Relayer) handleNonCallReq(f *Frame) error {
 	// Stop the timeout if the call if finished.
 	item, stopped, ok := items.Get(f.Header.ID, finished /* stopTimeout */)
 	if !ok {
-		return errUnknownID
+		return _frameNotDropped, errUnknownID
```

Comment:
this error case looks like the frame isn't forwarded, and so it is being dropped, shouldn't this be `_frameDropped`?